webapp/static/css/note-boards.css

```
@@ -263,3 +263,146 @@
+.board-modal .modal-content {
+  position: relative;
+  width: 100%; max-width: 520px; max-height: 78vh;
+  display: flex; flex-direction: column;
+  background: var(--card-bg, #fff);
+  color: var(--text-primary, #1f2933);
+  border: 1px solid var(--card-border, var(--glass-border, rgba(0, 0, 0, .12)));
+  border-radius: 16px;
+  box-shadow: 0 20px 50px rgba(0, 0, 0, .3);
+  animation: boardModalIn .22s ease;
+}
+@keyframes boardModalIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "boardModalIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)